src/main/java/edu/ksu/canvas/attendance/model/AttendanceSummaryModel.java

```diff
@@ -45,14 +48,16 @@ public Entry(long courseId, long sectionId, long studentId,
             this.totalClassesExcused = -1;
             this.totalClassesMissed = -1;
             this.totalClassesTardy = -1;
+            this.totalClassesPresent = -1;
         }
 
         public Entry(long courseId, long sectionId, long studentId,
-                     String studentName, boolean dropped, int totalClassesTardy, int totalClassesMissed, int totalClassesExcused) {
+                     String SisUserId, String studentName, boolean dropped, int totalClassesTardy, int totalClassesMissed, int totalClassesExcused, int totalClassesPresent) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00117)